web/client/src/library/components/documentation/Documentation.tsx

```diff
@@ -258,7 +261,12 @@ function DetailsItem<TValue = Record<string, Primitive>>({
     >
       {isArrayNotEmpty<TValue>(value) ? (
         <>
-          <strong className="mr-2 text-xs capitalize">{name}</strong>
+          <strong
+            title={title}
```

Comment:
What are these titles used for?

Reply:
when hovering over element, browser displays default tooltip with full name in this case